docs/docfx.json

```diff
@@ -34,7 +34,7 @@
     "overwrite": [],
     "externalReference": [],
     "globalMetadata": {
-      "uhfHeaderID": "MSDocsHeader-Dev_SharePoint",
+      "uhfHeaderID": "MSDocsHeader-M365-IT",
```

Comment:
Con you confirm this please? These are developer docs, so the other category seems like its correct. The content in this repo isn't IT Pro content.

Looked at the link you provided, but it doesn't say what options are available.

Reply:
This is a fair question -- the navigation structure and header implementation for the site has changed in the last couple of years since the previously listed header (Dev_SharePoint) was last supported, and this PR is to help get the SharePoint docs updated accordingly. In our current site IA, all offerings use their product family's header if they are part of a product family. Individual offerings that are not part of a product family will use their own header. Because SharePoint is branded as an M365 product, it needs to use the M365 product-level header. Please let me know if you have any other questions.